src/main/java/com/theelixrlabs/healthcare/dao/DoctorPatientAssignmentRepository.java

```diff
@@ -14,9 +15,17 @@ public interface DoctorPatientAssignmentRepository extends MongoRepository<Docto
     /**
      * Method declaration to find a DoctorPatientAssignmentModel by DoctorID and PatientID.
      *
-     * @param doctorID    UUID of the doctor associated with the assignment.
-     * @param patientID    UUID of the patient associated with the assignment.
+     * @param doctorId    UUID of the doctor associated with the assignment.
+     * @param patientId    UUID of the patient associated with the assignment.
      * @return Optional containing DoctorPatientAssignmentModel if found, empty otherwise.
      */
-    Optional<DoctorPatientAssignmentModel> findByDoctorIDAndPatientID(UUID doctorID, UUID patientID);
+    List<DoctorPatientAssignmentModel> findByDoctorIdAndPatientId(UUID doctorId, UUID patientId);
```

Comment:
Could you please explain this logic? you replaced the model object with the List

Reply:
Earlier, we thought the records will be unique in the database. So, I have taken Optional. But, later Shiju told to store all the records, so there can be more records of the same patientId and doctorId. So, I have taken a List.